### PR TITLE
fix(chat): fix image blinking (Issue #2272, #2347)

### PR DESCRIPTION
**Pull request by IlyaBondar**
**Description:**

fix image blinking

Issues:

- Issue #2272
- Issue #2347

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2378.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2378.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Chatbar/ModelIcon.tsx [hunk 1]
```diff
@@ -58,20 +66,16 @@ const ModelIconTemplate = memo(
...
-          <SVG
-            src={fallbackUrl}
-            width={size}
-            height={size}
-            description={description}
-          />
-        </SVG>
+          title={description}
+          onError={handleError}
+          alt=""
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Chatbar/ModelIcon.tsx]: is `alt` should be empty?

File: apps/chat/src/components/Chatbar/ModelIcon.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { memo } from 'react';
-import SVG from 'react-inlinesvg';
+/* eslint-disable @next/next/no-img-element */
+import { memo, useCallback, useRef } from 'react';
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Chatbar/ModelIcon.tsx:2]: I suppose we can uninstall react-inlinesvg package

**Review by Alexander-Kezik — DISMISSED** [review]

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2378.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by Alexander-Kezik — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2378.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2378.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — DISMISSED** [review]

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2378.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2378.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2378.nightly-test.deltixhub.io
>E2E tests status: success

**Review by irinakartun — APPROVED** [review]

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**